docs/source/api-reference/stylesheets/screen.css.scss

```diff
@@ -14,6 +14,17 @@ $govuk-assets-path: "/api-reference/assets/govuk/assets/";
   padding: 3px 0;
 }
 
+body.release-notes .technical-documentation {
+  h2 {
+    margin-bottom: 5px;
```

Comment:
happy to chat about this, but do we not want to keep the margin on h2 as is IF there are no tags? as now the text is quite close to the header?

Reply:
Ah yeah good point, I'll change it so the margin only appears for the headings with tags 👍